src/mmw/js/src/modeling/models.js

```diff
@@ -170,6 +192,29 @@ var ScenarioModel = Backbone.Model.extend({
 
         this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove', this.attemptSave, this);
+
+        var debouncedPoll = _.debounce(_.bind(this.pollForResults, this), 500);
+        this.get('modifications').on('add change', function() {
+            debouncedPoll();
+        });
+        this.set('taskModel', $.extend(true, {}, App.currProject.get('taskModel')));
```

Comment:
Now I get it. Consider adding a factory method on project to make this clearer `App.currentProject.createTaskModel()` instead of using extend. It probably doesn't make sense to have an instance of task model at the project level.